app/models/provider_interface/provider_applications_filter.rb

```diff
@@ -44,6 +44,7 @@ def no_results_message
   private
 
     def parse_params(params)
+      params[:candidate_name].strip! if params.has_key?(:candidate_name)
```

Comment:
I feel like I'm going crazy, there's got to be a less annoying way to do this?

Reply:
Maybe strip it in `FilterApplicationChoicesForProviders` instead? Then everyone gets the benefit

Reply:
Sorted, the data mangling now happens in the search itself, and avoids manipulating the original query the user put in. This also has the added benefit of affecting all the different bits of the app that use this search!

Reply:
I think we use a different service for support `SupportInterface::ApplicationsFilter`

Reply:
This should have been removed from the latest commit

Reply:
sorry not sure I follow, we just added the fix to the provider filter not the support filter, could we add it to the support filter as well?